### DISCUSSION TITLE
I am tying to train a LORA model on my 3060ti 8GB, 32GB RAM

### DISCUSSION THREAD

**Discussion by Odawgthat** [d8ahazard/sd_dreambooth_extension#887] [Q&A] [answered]
Hi, I have been trying to get dreambooth to run on my 8GB card for a while now and have come to the conclusion that the issue might be the verions of torch, cuda, etc... I have put a screenshot of my hash for stable diffusion etc, it always come up with a cuda out of memeory error even when I use silly settings like 20 steps on 2 images. 

![image](https://user-images.githubusercontent.com/122664491/216083620-595d50e6-ba51-47a3-88c7-6d994d81de05.png)

Hope someone can help me to figure out what the best versions are to run dreambooth, I heard some people saying that xformers v0.0.14dev0 was good which luckily I have, I also heard that cuda needs to be version 16? Again just anted to ask all of you guys. I have the latest version of dreambooth and stable diffusion.

**Comment by Shankyoz** [comment, marked as answer]
This is what I'm currently running, I do have a 3060ti,
with the new update for sd_ dreambooth extension , with the current xformers 0.016rc425 isn't training the models correctly
So I have had to use "flash attention" instead of xformers in the memory attention options, you have 14dev so you will be Ok with using xformers,

![curretly running](https://user-images.githubusercontent.com/124382742/216805319-1f273ba5-bacc-4990-afe7-1f3c3bfae580.PNG)

**Reply by Odawgthat** [reply]
In the end I just switched over to another webui for LORA training called "kohya_ss-master", it seems to do the job. Thanks for the input though, ill probably try and switch back to sd_dreambooth extension once they have ironed out the kinks...
